### PR TITLE
v1.24.0-beta.5 -> production

### PR DESCRIPTION
**Pull request by 0-vortex**
this is a re-release

### CODE REVIEW

File: supabase/diagrams/public.user_highlights.md [hunk 1]
```diff
@@ -4,16 +4,16 @@
...
-| ---------- | ------------------- | ------- | -------- | -------- | ------- | ------- |
-| id         | bigint              |         | false    |          |         |         |
-| user_id    | bigint              |         | false    |          |         |         |
-| url        | varchar(255)        |         | true     |          |         |         |
-| highlight  | varchar(500)        |         | false    |          |         |         |
-| created_at | time with time zone | now()   | false    |          |         |         |
-| updated_at | time with time zone | now()   | false    |          |         |         |
-| deleted_at | time with time zone |         | true     |          |         |         |
-| pinned     | boolean             | false   | false    |          |         |         |
+| Name       | Type                     | Default | Nullable | Children | Parents | Comment |
```
Comment (brandonroberts) [inline supabase/diagrams/public.user_highlights.md:7]: Why is this here? I haven't committed the db changes for highlights

Comment (0-vortex) [reply supabase/diagrams/public.user_highlights.md:7]: the action is checking the production db all the time, so we can't be publicly `0day`d :D
